### DISCUSSION TITLE
[v4] How to define decimal amounts in @theme?

### DISCUSSION THREAD

**Discussion by kerryj89** [tailwindlabs/tailwindcss#15117] [Help] [answered]
I have tried for example

```css
@import "tailwindcss";

@theme {
  --spacing-1.23: 1.23rem;
  --spacing-1\.23: 1.23rem;
  --spacing-1\\.23: 1.23rem;
}
```

And trying to apply it like `<span class="p-1.23">` but can't get it to work.  Non-decimal amounts do work for me, however.

**Comment by philipp-spiess** [comment, marked as answer]
@kerryj89 Hey! I agree that escaping dots should work where as they aren't right now. You can use underscores in the `@theme` block to work around this though: https://play.tailwindcss.com/nK0OAJ7prW

```css
@theme {
  --spacing-1_23: 1.23rem;
}
```

This works with the following HTML:

```html
<div class="p-1.23"></div>
```
